qasp/agile_bpa_qasp.md

```diff
@@ -0,0 +1,106 @@
+
+INTRODUCTION
+------------
+
+This Quality Assurance Surveillance Plan (QASP) has been developed to evaluate
+contractor actions while implementing the Performance Work Statement (PWS). It
+is designed to provide an effective surveillance method of monitoring contractor
+performance for each listed objective on the Performance Requirements Matrix in
+the call order. It also provides a systematic method to evaluate the services
+the contractor is required to furnish.
+
+STANDARD
+--------
+
+The contractor is responsible for management and quality control actions to meet
+the terms of the call order. The role of the COR/ACOR is quality assurance to
+ensure call order standards are achieved. The contractor shall perform all work
+required in a satisfactory manner in accordance with the requirements of the
+PWS. The COR/ACOR shall notify the CO for appropriate action if it is likely
+that the contractor will not achieve successful final delivery of the software
+code in accordance with the performance objectives and acceptable quality levels
+identified below.
+
+PERFORMANCE REQUIREMENTS MATRIX
+-------------------------------
+
+The COR/ACOR will evaluate the performance objectives through surveillance as
+reflected below by reviews and acceptance of work products and services. As
+indicated, the COR/ACOR will assess progress towards the final delivered
+software code. Note that the performance requirements listed below are required
+for the final deliverable. However, the sprints and incremental delivery of code
+will be assessed by the Government to ensure that the contractor is on a path to
+successful final delivery.
+
+Deliverable or Required Services Performance Standard(s):
+
+  **Deliverable or Required Services** | **Performance Standard(s)** | **Acceptable Quality Level** | **Method of Surveillance**
+   ----------- | ---------------- | -------------- | ---------------
+   Tested Code | Code delivered under the order must have substantial test code coverage and a clean code base. | Minimum of 90% test coverage of all relevant code | Combination of manual review and automated testing, using agreed-upon publicly-available SaaS products
+   Accessible | Client-side rendering must conform with section 508 standards and WCAG2AA standards. | 0 errors reported for WCAG2AA using an automated scanner, 0 errors reported in manual testing, and all warnings confirmed as non-issues |  http://squizlabs.github.io/HTML_CodeSniffer/ or https://github.com/pa11y/pa11y
+   Deployed | Code must successfully build and deploy into staging environment. | Successful build with a single command | Combination of manual review and automatic testing
+   Documented | All dependencies (and licenses for dependencies) are listed and all major functions are documented. | All dependencies are listed and the licenses are documented. Software/source code is documented. System diagram is provided. | Combination of manual review and automatic testing
```

Comment:
Do we want to specify how? Documented suggests that this could be heavy, is list i.e. package.json/gemfile/requirements.txt or are you looking for open control here?